cmd/publish.go

```diff
@@ -0,0 +1,155 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/avorty/spito/cmd/cmdApi"
+	"github.com/avorty/spito/internal/checker"
+	"github.com/avorty/spito/pkg/shared"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	publishRulesetRoute = "ruleset/publish"
+)
+
+type RuleForRequest struct {
```

Comment:
I think it would be good idea to rename `RuleForRequest` into `RuleConfig` and put it into `checker` package. (Actually I'm not sure whether it isn't already similar struct in `checker` package)